src/common-lib/constants.ts

```diff
@@ -1,7 +1,32 @@
+import { IN_DEVELOPMENT } from '../config/env';
+
 export enum ENTRANCE {
   ADMIN = 'circle-create-initial-admin',
   LINK = 'magic-link',
   MANUAL = 'manual-address-entry',
   CSV = 'CSV',
   NOMINATION = 'vouched-in',
 }
+
+let chains: { [key: number]: string } = {
+  1: 'Ethereum Mainnet',
+  10: 'Optimism',
+  100: 'Gnosis',
+  137: 'Polygon',
+  250: 'Fantom Opera',
+  42220: 'Celo Mainnet',
+  42161: 'Arbitrum One',
+  43114: 'Avalanche C-Chain',
+  1313161554: 'Aurora Mainnet',
+};
+if (IN_DEVELOPMENT) {
```

Comment:
I don't think there's any harm in allowing Goerli on production

Reply:
Vaults code already makes sure the data it's fetching corresponds to the current chain (it checks `vault.chain_id`)

Reply:
Sure, no harm, but desirable as a feature? @levity

Reply:
It would let people test-drive vaults on Goerli

Reply:
ok, added Goerli to main section.